cryptarchia/cryptarchia.py

```diff
@@ -30,27 +33,34 @@ class Config:
     k: int  # The depth of a block before it is considered immutable.
     active_slot_coeff: float  # 'f', the rate of occupied slots
 
-    # The stake distribution is always taken at the beginning of the previous epoch.
+    # The stake distribution is taken at the beginning of the previous epoch.
     # This parameters controls how many slots to wait for it to be stabilized
-    # The value is computed as epoch_stake_distribution_stabilization * int(floor(k / f))
+    # The value is computed as
+    #     epoch_stake_distribution_stabilization * int(floor(k / f))
     epoch_stake_distribution_stabilization: int
-    # This parameter controls how many slots we wait after the stake distribution
-    # snapshot has stabilized to take the nonce snapshot.
+    # This parameter controls how many `base periods` we wait after stake
+    # distribution snapshot has stabilized to take the nonce snapshot.
     epoch_period_nonce_buffer: int
-    # This parameter controls how many slots we wait for the nonce snapshot to be considered
-    # stabilized
+    # This parameter controls how many `base periods` we wait for the nonce
+    # snapshot to be considered stabilized
     epoch_period_nonce_stabilization: int
 
+    # -- Stake Relativization Params
+    initial_total_stake: int  # D_0
+    total_stake_learning_rate: int  # beta
```

Comment:
this is a bit misleading? should it be total_active_stake?

Reply:
good point, let me rename these fields